crates/core/tedge/src/cli/connect/bridge_config_azure.rs

```diff
@@ -59,8 +59,13 @@ impl From<BridgeConfigAzureParams> for BridgeConfig {
             topics: vec![
                 pub_msg_topic,
                 sub_msg_topic,
+                // Direct methods (request/response)
+                r##"topic methods/POST/# in 1 az/ $iothub/"##.into(),
+                r##"topic methods/res/# out 1 az/ $iothub/"##.into(),
+                // Digital twin
```

Comment:
I would add a comment pointing to https://learn.microsoft.com/en-us/azure/iot/iot-mqtt-connect-to-iot-hub#receiving-cloud-to-device-messages, the topic names being not so clear.

Reply:
resolved